### PR TITLE
Some update & Fix

### PR DESCRIPTION
**Pull request by Aerue**
examineSeachInfo is now working !

- classjob data are inverted (id <=> lvl) compared to saphire comment
- first set of classJob is wrong
- dnc is missing

### CODE REVIEW

**Comment by Aerue** [comment]
Updated missing history entries in marketBoardItemListingHistories @karashiiro

File: models/examineSearchInfo.js [hunk 1]
```diff
@@ -50,3 +57,46 @@ const getWorld = (id) => { // TODO: use World exd
...
+    if (id ===  31) return "NIN";
+    if (id ===  32) return "MCH";
+    if (id ===  33) return "DRK";
+    if (id ===  34) return "AST";
+    if (id ===  35) return "SAM";
+    if (id ===  36) return "RDM";
+    if (id ===  37) return "BLU";
+    if (id ===  38) return "GNB";
+
+    // DNC is missing ?
```
Comment (karashiiro) [inline models/examineSearchInfo.js]: DNC is ID 38, these all need to be shifted back one so that DNC can be included.

Comment (karashiiro) [reply models/examineSearchInfo.js]: From my own data, for reference: `37,0,80,0,38,0,70,0`. I have GNB (37) at 80, and DNC (38) at 70.

File: models/examineSearchInfo.js [hunk 2]
```diff
@@ -50,3 +57,46 @@ const getWorld = (id) => { // TODO: use World exd
     if (id ===  97) return "Ragnarok";
     if (id ===  99) return "Sargatanas";
 };
+const getJob = (id) => {
+    if (id ===  1) return ""; // ? Unknow
```
[the remark this answers is not in the document]
Comment (karashiiro) [reply models/examineSearchInfo.js]: Err, no, that's not quite right either. Just checked and got `1,11,0,0` there. That means this dummy job would have an ID of 267 as a uint16_t. This unknown probably isn't a job at all.

**Comment by Aerue** [comment]
Should be OK now, i was tired when i found that DNC was missing

**Comment by karashiiro** [comment]
Thank you!

**Pull request merged**